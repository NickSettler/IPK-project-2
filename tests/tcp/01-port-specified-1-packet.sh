./ipk-sniffer -i en0 --tcp -p 22