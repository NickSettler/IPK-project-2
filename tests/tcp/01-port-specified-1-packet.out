timestamp: 2023-04-17T19:50:36.270+02:00
src MAC: b0:be:83:4a:c4:b0
dst MAC: 98:77:e7:50:60:5e
frame length: 134 bytes
src IP: 192.168.1.17
dst IP: 147.229.176.14
src port: 64168
dst port: 22

0x0000: 98 77 e7 50 60 5e b0 be 83 4a c4 b0 08 00 45 00  .w.P`^...J....E.
0x0010: 00 78 00 00 40 00 40 06 34 d3 c0 a8 01 11 93 e5  .x..@.@.4.......
0x0020: b0 0e fa a8 00 16 71 3c 60 c4 c8 0c 4d 8e 80 18  ......q<`...M...
0x0030: 08 00 95 5a 00 00 01 01 08 0a ab 89 dd ea c8 1e  ...Z............
0x0040: aa 97 e4 1b 57 a6 a9 3f 26 45 ce 1b ac 94 fb 8a  ....W..?&E......
0x0050: f6 3a c5 de 08 f1 c2 16 c5 cb bd 2e 1a 4b c3 df  .:...........K..
0x0060: c8 3d f7 b4 80 4f cf 1e 6b b0 99 a1 ea cd c2 1c  .=...O..k.......
0x0070: e7 28 db 9f b2 b0 63 fe e9 65 97 8c ea f5 43 9a  .(....c..e....C.
0x0080: b1 44 fc 1e 93 1e                                .D....

